components/typography/Paragraph.js

```diff
@@ -0,0 +1,23 @@
+
+export default function Paragraph({
+    typeStyle = "body-lg",
+    textColor = "text-gray-700",
+    className,
+    children
+}) {
+
+    let classNames
```

Comment:
That same situation as in `Heading` component, please default that var to the `""`.

Reply:
Done!